packages/sdk/src/sdk/services/Auth/UserAuth.ts

```diff
@@ -138,8 +139,13 @@ export class UserAuth implements AuthService {
     })
   }
 
-  hashAndSign: (data: string) => Promise<string> = () => {
-    throw new Error('hashAndSign not initialized')
+  hashAndSign: (data: string) => Promise<string> = async (data) => {
```

Comment:
Just FYI `UserAuth` isn't currently used in Client. I'm planning on updating things soon to use this class. But if this is crucial to fixing anything right now, there's another spot that needs to be updated.